src/python/doc/wasserstein_distance_user.rst

```diff
@@ -100,6 +100,57 @@ The output is:
     point 2 in dgm1 is matched to the diagonal
     point 1 in dgm2 is matched to the diagonal
 
+
+Entropic regularization
+***********************
+
+When using the Optimal Transport based version of wasserstein distances, 
+we also allow for the use of entropic regularization of optimal transport distance via the parameter ``reg``.
+If ``reg > 0``, we solve 
+:math:`\argmin_P \sum P_{ij} M_{ij} + reg * h(P)`
```

Comment:
I am seeing the `\` before argmin in the sphinx doc.

Reply:
ah yes my bad, I am so used to defined `argmin` as a macro that I forgot it wasn't by default, will fix this.